Итоговый проект/FileManager/Commands/DataGridCommands.cs

```diff
@@ -0,0 +1,350 @@
+        });
+        public static readonly ICommand PasteObject = new RelayCommand(o =>
+        {
+            var PathString = GetPath();
+            var SourcePathString = Clipboard.GetText();
+
+            try
+            {
+                if (Directory.Exists(SourcePathString))
+                {
+                    var DirInfo = new DirectoryInfo(SourcePathString);
+                    var FullPath = GetPath(DirInfo.Name);
+
+                    if (Directory.Exists(GetPath()) && !Directory.Exists(FullPath))
+                    {
+                        DirCopy(SourcePathString, FullPath);
+                        if (IsCut)
+                        {
+                            DirDelete(SourcePathString);
+                            IsCut = false;
+                        }
+                    }
+                }
+                else if (File.Exists(SourcePathString))
+                {
+                    var FullPath = Path.GetFileName(SourcePathString);
+
+                    if (Directory.Exists(GetPath()) && !File.Exists(GetPath(FullPath)))
+                    {
+                        File.Copy(SourcePathString, FullPath);
+                        if (IsCut)
+                        {
+                            File.Delete(SourcePathString);
+                            IsCut = false;  
+                        }
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message);
+            }
+            finally
+            {
+                RefreshDataGrid();
+            }
+        });
+        public static readonly ICommand DeleteObject = new RelayCommand(o =>
+        {
+            var PathString = GetPath(GetDataForContextMenu((Border)o).Text);
+
+            if (o is Border)
+            {
+                try
+                {
+                    if (Directory.Exists(PathString))
+                    {
+                        DirDelete(PathString);
+                    }
+                    else if (File.Exists(PathString))
+                    {
+                        File.Delete(PathString);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    MessageBox.Show(ex.Message);
+                }
+                RefreshDataGrid();
+            }
+        });
+        public static readonly ICommand CreateFile = new RelayCommand(o =>
+        {
+            var NewModalWindow = new CreateObjectWindow(true, Main.TextBoxPath.Text);
+            NewModalWindow.Owner = Main;
+            NewModalWindow.Title = "Создать файл";
+            NewModalWindow.ShowDialog();
+            if (NewModalWindow.DialogResult == true)
+            {
+                var FullPath = NewModalWindow.GetFullPath();
+                if (!File.Exists(FullPath))
+                {
+                    try
+                    {
+                        File.Create(FullPath);
```

Comment:
```suggestion
                        File.Create(FullPath).Dispose();
```
Возвращается отрытый поток!